PowerPointLabs/PowerPointLabs/ELearningLab/Views/ELearningWorkspaceViews/ELearningLabMainPanel.xaml.cs

```diff
@@ -799,12 +806,15 @@ private bool DoesSlideExist(int slideId)
             return false;
         }
 
-        private void SetupSpinnerOnLoadingBegins()
+        private void SetupSpinnerOnLoadingBegins(string content)
```

Comment:
Maybe the name `Begin` would be more consistent

Reply:
Thanks! Renamed method name to `...Begin`